### DISCUSSION TITLE
Migrate mysql server permission issues

### DISCUSSION THREAD

**Discussion by NSOPORTEDESARROLLO** [LiveHelperChat/livehelperchat#2166] [Q&A]
Hi, I'm trying dump mysql and restore on new server, using mysql versions ( old server: 8.0.26 , new server: 8.0.32). Everything fine but I 'm getting "You do not have permission to access the current chat or chat was merged/deleted!"  when open any chat ( closed, pending) and using administrator user.

- I'm not upgrading
- Using mysqldump -p database > database.sql.     mysql -p database < database.sql

**Comment by remdex** [comment]
That has nothing to do with mysql migration. Reasons can he numereus. Refer to docs...

https://doc.livehelperchat.com/docs/chat/#permissions-influencing-chat-opening-permissions

**Reply by NSOPORTEDESARROLLO** [reply]
Hi @remdex, I know that migration shouldn't be a problem, but it seems that something is not working right. I copied the mysql folder from the old server to the new one and everything works fine, the problem is in the mysqldump. I will review the problem and let you know in case it happens to someone else.
